tests/test_player_with_strategy_one.py

```diff
@@ -9,18 +8,50 @@ def setUp(self):
         self.player_blue = PlayerWithStrategyOne('blue')
         self.player_red = PlayerWithStrategyOne('red')
 
-    def test_player_wins_with_strategy_one(self):
-        self.board_player_blue_wins = GameBoard.from_matrix([
-                    ['blue', 'blue', 'blue',  'red',   None,   None, None],
-                    ['red',   'red',  'red', 'blue', 'blue',   None, None], # <- blue row wins
-                    ['blue',  'red', 'blue',  'red', 'blue',   None, None],
-                    ['blue',  'red', 'blue',  'red',  'red',  'red', None],
-                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', None],
-                    ['red',   'red',  'red', 'blue',  'red', 'blue', None]])
-        self.match = Match(self.player_blue, self.player_red, self.board_player_blue_wins)
-        self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        for i in range(8):
-            self.match.play_next_turn()
-            self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        self.match.play_next_turn()
-        self.assertTrue(self.player_blue.is_winner(self.board_player_blue_wins))
+    def test_player_with_strategy_one_blue_wins_on_right_upper_corner(self):
+        board = GameBoard.from_matrix([
+                    ['red',  'blue',  'red',  'red', 'blue',  'blue',   None],
+                    ['red',   'red', 'blue',  'red',  'red',  'red', 'blue'],
+                    ['red',  'blue',  'red',  'red', 'blue', 'blue', 'blue'],
+                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', 'blue'],
+                    ['red',  'blue',  'red', 'blue',  'red', 'blue',  'red'],
+                    ['blue',  'red', 'blue',  'red',  'red',  'red',  'red']])
+        self.assertFalse(self.player_blue.is_winner(board))
+        self.player_blue.play(board)
+        self.assertTrue(self.player_blue.is_winner(board))
```

Comment:
I wonder why are you testing against `is_winner` instead of checking on the board (e.g. that there is a blue chip on the upper right corner). While in this case it would be the same, in the tests below it is not (you assert that the player is no winning, but you have no idea where the chip is).